tests/integration/config/test_validate.py

```diff
@@ -63,7 +63,10 @@ def test_parse_configuration_transforms_file_into_mapping():
     config, logs = module.parse_configuration('/tmp/config.yaml', '/tmp/schema.yaml')
 
     assert config == {
-        'location': {'source_directories': ['/home', '/etc'], 'repositories': ['hostname.borg']},
+        'location': {
+            'source_directories': ['/home', '/etc'],
+            'repositories': [{'path': 'hostname.borg', 'label': ''}],
```

Comment:
Wouldn't it make more sense to omit `label` if it's not set? After all, it's optional. Similar throughout this test file.